src/main/java/seedu/address/model/commons/ModuleCode.java

```diff
@@ -25,7 +25,7 @@ public class ModuleCode {
     public ModuleCode(String moduleCode) {
         requireNonNull(moduleCode);
         checkArgument(isValidModule(moduleCode), MESSAGE_CONSTRAINTS);
-        this.moduleCode = moduleCode;
+        this.moduleCode = moduleCode.toUpperCase();
```

Comment:
`toUpperCase()` in already handled in `src/main/java/seedu/address/logic/parser/ParserUtil.java`
More info: #224

Reply:
I think need to handle both because if `parserUtil` only handles user input,  it doesn't handle when the user edits the data file.

Reply:
And I think it is better to use `parseVenue` and `parseModuleCode` method in `src/main/java/seedu/address/logic/parser/ParserUtil.java` when parsing consultation

Reply:
Got it!